src/drivers/angular2.js

```diff
@@ -0,0 +1,99 @@
+import { omit, replaceObject } from '../lib';
+/* eslint-disable new-cap, object-shorthand */
+
+
+export let angular2 = {
+
+    global() {
```

Comment:
We just need to check how that impacts angular's ahead of time compilation. One more thing for you to not like in angular :)